docs/developers/contribute.md

```diff
@@ -1,9 +1,17 @@
 # Contribute
 
-Welcome to the molfeat community! We appreciate everyone's contribution and welcome all. Apart from code contributions, there are various other ways to support the community such as answering questions, providing assistance to others, enhancing the documentation, giving a ⭐️ or sharing the project.
+Welcome to the molfeat community! We appreciate everyone's contribution and welcome all. Apart from code contributions, there are various other ways to support the community. In short, you can: 
+
+- :star: **Star** the repository on Github
```

Comment:
This seems like a duplicate of the *ways to contribute section* below. Would likely make sense to pull that section up here and/or rephrase both a bit